.github/workflows/build_images_and_publish.yaml

```diff
@@ -0,0 +1,124 @@
+name: 'Build Images and Publish'
```

Comment:
The reason why I can only execute this on my fork is, that we have not yet set up repo secrets in the main EDC repo. 
However, like stated above, I think we can easily remove that workflow altogether, because I just created it as a convenience feature for the hackathon.